Cargo.toml

```diff
@@ -29,11 +29,14 @@ travis-ci = { repository = "dalek-cryptography/x25519-dalek", branch = "master"}
 features = ["nightly"]
 
 [dependencies]
-curve25519-dalek = { version = "1", default-features = false }
+curve25519-dalek = { version = "2.0.0-alpha.0", default-features = false }
 rand_core = { version = "0.3", default-features = false }
 clear_on_drop = { version = "0.2" }
+our_serde = { package = "serde", version = "1", default-features = false, optional = true, features = ["derive"] }
```

Comment:
There should probably be a comment in the `Cargo.toml` explaining why `serde` is imported as `our_serde` -- because we need to enable the `curve25519-dalek/serde` feature, this crate needs a feature that enables both the `curve25519-dalek/serde` feature and the `serde` dependency, and renaming the dependency avoids a name collision.